config/services.yml

```diff
@@ -4,13 +4,13 @@ parameters:
   # Environment variables for dynamic configuration override:
   env(ORCA_AMPLITUDE_API_KEY): ~
   env(ORCA_AMPLITUDE_USER_ID): ~
-  env(ORCA_COVERAGE_CLOVER): "%kernel.project_dir%/var/coverage/clover.xml"
+  env(ORCA_COVERAGE_CLOVER): "%app.project_dir%/var/coverage/clover.xml"
   env(ORCA_FIXTURE_DIR): "%app.fixture_dir%"
-  env(ORCA_JUNIT_LOG): "%kernel.project_dir%/var/junit/junitLog.xml"
+  env(ORCA_JUNIT_LOG): "%app.project_dir%/var/junit/junitLog.xml"
   env(ORCA_PACKAGES_CONFIG): config/packages.yml
   env(ORCA_PACKAGES_CONFIG_ALTER): ~
   env(ORCA_PHPCS_STANDARD): "AcquiaDrupalTransitional"
-  env(ORCA_TELEMETRY_ENABLE): false
+  env(ORCA_TELEMETRY_ENABLE): "false"
```

Comment:
Did you check to make sure our environment variable reading functionality handles this identically? (In the absence of the variable on the shell and in its presence--with both `true` and `false` values?

Reply:
I have checked with the following combinations in my local-
- export ORCA_TELEMETRY_ENABLE=true
- export ORCA_TELEMETRY_ENABLE=false
- unset ORCA_TELEMETRY_ENABLE

All these behave in the same way before and after the change